pkgs/applications/networking/bwsyncandshare/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, oraclejre8, ... }:
+
+stdenv.mkDerivation rec {
+  name = "bwSyncAndShare-${version}";
+  version = "11.6.703";
+
+  src = fetchurl {
+    # No version-numbered tarball available, only a "Latest" version.
+    # This will break whenever a new version is published.
+    url = https://download.bwsyncandshare.kit.edu/clients/bwSyncAndShare_Latest_Linux.tar.gz;
```

Comment:
I think the policy is to not include packages that don't have stable sources.

That said, their [copyright page](https://bwsyncandshare.kit.edu/imprint/imprint.html) says:

> Weiterverbreitung, auch in Auszügen, für pädagogische, wissenschaftliche oder private Zwecke ist unter Angabe der Quelle gestattet (sofern nicht anders an der entsprechenden Stelle ausdrücklich angegeben).

I think this means that we can mirror the package ourselves, and mark it as `licenses.unfreeRedistributable` instead.

Reply:
@yegortimoshenko thank you. Unfortunately the LICENSE file included with the tarball clearly prohibits redistribution.  I will ask them to provide stable download links. Closing this until they do.